Lesson-3/src/Lesson3.java

```diff
@@ -0,0 +1,97 @@
+public class Lesson3 {
+
+    public static void printThreeWords (){
+        System.out.println("Orange\nBanana\nApple");
+    }
+
+    public static void checkSumSign (int a, int b){
+
+        if ((a+b) >= 0){
```

Comment:
Эти скобки лишнее, просто нет смысла в них.
```suggestion
        if (a+b >= 0){
```